src/main/java/org/codehaus/plexus/archiver/jar/JdkManifestFactory.java

```diff
@@ -33,25 +33,33 @@
     public static java.util.jar.Manifest getDefaultManifest()
         throws ArchiverException
     {
-        final java.util.jar.Manifest defaultManifest = new java.util.jar.Manifest();
-        defaultManifest.getMainAttributes().putValue( "Manifest-Version", "1.0" );
+        try
+        {
+            final java.util.jar.Manifest defaultManifest = new java.util.jar.Manifest();
+            defaultManifest.getMainAttributes().putValue( "Manifest-Version", "1.0" );
 
-        String createdBy = "Plexus Archiver";
+            String createdBy = "Plexus Archiver";
 
-        InputStream inputStream = JdkManifestFactory.class.getResourceAsStream( "/META-INF/"
-            + "maven/org.codehaus.plexus/plexus-archiver/pom.properties" );
-        Properties properties = PropertyUtils.loadProperties( inputStream );
-        if ( properties != null )
-        {
-            String plexusArchiverVersion = properties.getProperty( "version" );
-            if ( plexusArchiverVersion != null )
+            final Properties properties = PropertyUtils.loadProperties( JdkManifestFactory.class.getResourceAsStream(
+                "/META-INF/maven/org.codehaus.plexus/plexus-archiver/pom.properties" ) );
+
+            if ( properties != null )
             {
-                createdBy += " " + plexusArchiverVersion;
+                String plexusArchiverVersion = properties.getProperty( "version" );
+                if ( plexusArchiverVersion != null )
+                {
+                    createdBy += " " + plexusArchiverVersion;
+                }
             }
-        }
-        defaultManifest.getMainAttributes().putValue( "Created-By", createdBy );
 
-        return defaultManifest;
+            defaultManifest.getMainAttributes().putValue( "Created-By", createdBy );
+
+            return defaultManifest;
+        }
+        catch ( final IOException e )
+        {
```

Comment:
Am 05/27/16 um 00:38 schrieb Michael Osipov:
> In
> src/main/java/org/codehaus/plexus/archiver/jar/JdkManifestFactory.java
> <https://github.com/codehaus-plexus/plexus-archiver/pull/41#discussion_r64833184>:
> 
>>  
>> -        return defaultManifest;
>> +            defaultManifest.getMainAttributes().putValue( "Created-By", createdBy );
>> +
>> +            return defaultManifest;
>> +        }
>> +        catch ( final IOException e )
>> +        {
> 
> What purpose serves this try catch block? No one throws an
> |IOException|. Not the |PropertyUtils|.

PropertyUtils started throwing the exception with the latest patch
release of 'plexus-utils'.